app/styles/component/_site-head.scss

```diff
@@ -133,6 +143,25 @@ body:not(.no-scroll) {
 	}
 }
 
+.site-head__partner-slot {
+	flex-basis: 100%;
+}
+
+.site-head__partner-logo {
+	left: 50%;
+	position: absolute;
```

Comment:
do we need to use absolute positioning? flexbox won't work?

Reply:
flex works, but then it is too low and the whole navbar is higher than it needs to be